src/test/pages/Registration.test.tsx

```diff
@@ -20,9 +20,7 @@ describe("Registration", () => {
     // Locate the radio button for teacher or student and click on it
     const radioButton = screen.getByLabelText(option);
     await userEvent.click(radioButton);
-    // Locate the "Submit" button and click on it
-    const submitButton = screen.getByRole("button", { name: "Continue" });
-    await userEvent.click(submitButton);
+    await clickButton("Continue");
```

Comment:
It looks like this test is divided in blocks, each block starting with a comment. Since you've refactored this bit and your bit is the only one without a comment now, this looks like it's part of the previous block (`// Locate the radio button for teacher or student and click on it`). You have a couple of options: you could add a comment above this line, or you could add a couple of empty lines before and after this one, so that it's clear that this line is its own block. Does that make sense?

The same goes for line 39.

Since you're reading Clean Code, this may be familiar, as it is one example of a comment telling a lie :-)